influxdb3_wal/src/lib.rs

```diff
@@ -525,6 +527,28 @@ pub struct MetaCacheDelete {
     pub cache_name: Arc<str>,
 }
 
+#[derive(Debug, Serialize, Deserialize, Eq, PartialEq, Clone)]
+pub struct PluginDefinition {
```

Comment:
these types all seem the same as the ones above? Can we just have one set of types and share them between the WAL and the plugin crate?